pycbc/waveform/utils.py

```diff
@@ -44,6 +44,65 @@ def ceilpow2(n):
         exponent -= 1;
     return (1) << exponent;
 
+def coalign_waveforms(h1, h2, psd=None,
+                      low_frequency_cutoff=None,
+                      high_frequency_cutoff=None,
+                      resize=True):
+    """ Return two time series which are aligned in time and phase.
+
+    The alignment is only to the nearest sample point and all changes to the
+    phase are made to the first input waveform. Waveforms should not be split
+    accross the vector boundary. If it is, please use roll or cyclic time shift
+    to ensure that the entire signal is contiguous in the time series.
+
+    Parameters
+    ----------
+    h1: pycbc.types.TimeSeries
+        The first waveform to align.
+    h2: pycbc.types.TimeSeries
+        The second waveform to align.
+    psd: {None, pycbc.types.FrequencySeries}
+        A psd to weight the alignment
+    low_frequency_cutoff: {None, float}
+        The low frequency cutoff to weight the matching in Hz.
+    high_frequency_cutoff: {None, float}
+        The high frequency cutoff to weight the matching in Hz.
+    resize: Optional, {True, boolean}
+        If true, the vectors will be resized to match each other. If false,
+        they must be the same length and even in length
+
+    Returns
+    -------
+    h1: pycbc.types.TimeSeries
+        The shifted waveform to align with h2
+    h2: pycbc.type.TimeSeries
+        The shifted waveform to align with h1
```

Comment:
This waveform isn't shifted at all though right? It might be resized as you say but not shifted? Maybe just change the docstring to reflect this and then I'm happy for this to be merged.

Reply:
waveform `h2` I mean